tests/Controller/ProfileControllerTest.php

```diff
@@ -21,6 +31,13 @@ public function setUp(): void
         parent::setUp();
     }
 
+    public function tearDown(): void
+    {
+        static::getContainer()->get(Connection::class)->rollBack();
+
+        parent::tearDown();
+    }
+
```

Comment:
this is pretty easy to forget and would make a good case for a nice sideffect on the next functional test.

I would suggest we base class some stuff in the functional tests into a ControllerTestCase and let all Controller tests extend from this?  
Would allow to introduce a getEM() on tests as well

Would of course do this in another PR.

Reply:
Yeah sounds reasonable